### DISCUSSION TITLE
Checkbox Size

### DISCUSSION THREAD

**Discussion by air1rxk** [zim-desktop-wiki/zim-desktop-wiki#2022] [Q&A] [answered]
Anyway to change the checkbox size? Is this Zim coded or coming from themes? Checkbox doe not seem to change with font size change,

**Comment by introt** [comment]
Apart from modifying the underlying pixmaps, afaik not currently; the checkboxes are implemented as small, non-scalable images.

As a workaround, you can use your desktop's general scaling options instead of changing the text size.

**Comment by bruvellu** [comment, marked as answer]
You can change the checkbox size by adding the `bullet_icon_size` parameter to the `[TextView]` in `style.conf`. Use one of the available options of [GTK's IconSize](https://docs.gtk.org/gtk3/enum.IconSize.html). These are fixed sizes and won't scale with font size changes. See #181 or the example below:

```
[TextView]
bullet_icon_size=ICON_SIZE_LARGE_TOOLBAR
```

**Reply by air1rxk** [reply]
Thank You nelas for quick response... I used the "bullet_icon_size=ICON_SIZE_DND" and got the size i was looking for. Just started using Zim and Github still learning. I used # to comment out the existing line i hope ok seemed to work. Wanted reference if i decide to change back. How do you get that neat copy box for text in your reply?

**Reply by introt** [reply]
Yes, commenting lines out is ok.

> How do you get that neat copy box for text in your reply?

See here: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code
